src/api/sso/test/e2e/auth-flows.test.js

```diff
@@ -4,14 +4,8 @@ const { hash } = require('@senecacdot/satellite');
 // We need to get the URL to the sso service running in docker, and the list
 // of allowed origins, to compare with assumptions in the tests below.
 const { SSO_URL, ALLOWED_APP_ORIGINS } = process.env;
-const {
-  login,
-  logout,
-  getTokenAndState,
-  createTelescopeUsers,
-  cleanupTelescopeUsers,
-  ensureUsers,
-} = require('./utils');
+const { login, logout, getTokenAndState } = require('./browserUtil');
```

Comment:
`browser-util` please, don't rely on case in the filesystem, as not all filesystems are case sensitive.